Func.Isolated.Net7.With.AI/appsettings.json

```diff
@@ -1,10 +1,21 @@
 ﻿{
   "Logging": {
     "LogLevel": {
-      "Default": "Information",
       "Func.Isolated.Net7.With.AI.MyOtherFunctions": "Error",
       "Func.Isolated.Net7.With.AI.MyUserFunctions": "Debug",
-      "Func.Isolated.Net7.With.AI.UserDataService": "Trace"
+      "Func.Isolated.Net7.With.AI.UserDataService": "Trace",
+      "Azure.Core": "Error",
```

Comment:
Good idea to add Azure.Core

Reply:
I have not tested this though, but my understanding is, that this should be already filtered by the Azure Functions Worker through host.json, but better safe than sorry

Reply:
But that's not ideal. We need to know how it works. That is the whole reason why I created this sample so everything is clear. I use it now as boilerplate for new projects. I try to minimize preview packages and unknown configurations until Microsoft reacts with a valid supported sample application.

Reply:
Yeah i know it is not ideal, but at least it's working :-/ I am also really unhappy in configure so many loglevels and using this preview packages. It is really discouraging that Microsoft did not react so far to clarify any of this.